Elastiflix/dotnet-login-elastic-auto/Dockerfile

```diff
@@ -1,3 +1,7 @@
+##
```

Comment:
I went through several source files in this repo and couldn't find any license headers, which seems correct for an examples repo. Should we add these headers?

Reply:
I can remove the license headers. No problem